Bento/Bento/UITableViewExtensions.swift

```diff
@@ -0,0 +1,60 @@
+extension UITableView {
+    public func prepareForBoxRendering<SectionId: Hashable, RowId: Hashable>(
+        sectionIdType: SectionId.Type,
+        rowIdType: RowId.Type
+    ) {
+        prepareForBoxRendering(BentoTableViewAdapter<SectionId, RowId>.self)
+    }
+
+    public func prepareForBoxRendering<SectionId, RowId>(
+        _ type: TableViewAdapter<SectionId, RowId>.Type
```

Comment:
Would it be nicer to path instance here ? 🤔

Reply:
Hmm.

Reply:
This does. You specify the type of the adapter, and Bento would create it internally.

Reply:
Both https://github.com/Babylonpartners/babylon-ios/pull/5317/ and 
[one of the examples](https://github.com/Babylonpartners/Bento/blob/56c4c6f310cbd227f9ea214f032cecb94ba871e5/Example/BookAppointmentViewController.swift) are using it.

Reply:
I believe it won't work if client has custom initializer